doc/command-line.md

```diff
@@ -247,38 +251,106 @@ Usage:
 sourmash lca --query query.sig [query2.sig ...] --db <lca db> [<lca db2> ...]
```

Comment:
classify is left out:
sourmash lca classify --query query.sig [query2.sig ...] --db <lca db> [<lca db2> ...]

Reply:
good catch!